src/networking.c

```diff
@@ -537,12 +537,18 @@ void addReplyError(client *c, const char *err) {
     afterErrorReply(c,err,strlen(err));
 }
 
+void addReplyErrorSdsExt(client *c, sds err, int flags) {
+    addReplyErrorLength(c,err,sdslen(err));
+    if (!(flags & ERR_REPLY_FLAG_NO_STATS_UPDATE)) {
+        afterErrorReply(c,err,sdslen(err));
```

Comment:
i think we still need to call `afterErrorReply`, and pass that flag forward

Reply:
Fixed